management-app/src/api/types.ts

```diff
@@ -0,0 +1,12 @@
+export type TAPIErrorsMap = {
+  [key: number]: string;
+};
+
+export type TUser = TUserBase & {
```

Comment:
TUserBase should be declared before it's used for a better code readability

Reply:
nice catch